test/SILOptimizer/string_switch.swift

```diff
@@ -1,6 +1,7 @@
 // RUN: %target-build-swift -O %s -module-name=test -Xllvm -sil-disable-pass=FunctionSignatureOpts -o %t.out
 // RUN: %target-build-swift -O %s -module-name=test -Xllvm -sil-disable-pass=FunctionSignatureOpts -emit-sil | %FileCheck %s
 // RUN: %target-run %t.out
+// REQUIRES: swift_stdlib_no_asserts,optimized_stdlib
```

Comment:
Is this still necessary?

Reply:
I'm not sure, I'll make sure the tests pass with this, then I can try taking it out again.